### ISSUE TITLE
Title UndefinedUndefined

### ISSUE THREAD

**Issue by jfillipe** [fullcalendar/fullcalendar#3911]
I´ve been updated, and i get this.
All view´s i get this.

![image](https://user-images.githubusercontent.com/28539354/32408214-727fa2a4-c17b-11e7-9241-5aae25a83995.png)

Any ideas?

Thanks guys =)

**Comment by houseoftech** [comment]
I'm seeing the same thing. It's happening between upgrade from 3.4.0 to 3.5.0.

**Comment by arshaw** [comment]
Would you be able to post a [runnable, stripped-down demonstration of the bug](http://fullcalendar.io/wiki/Reduced-Test-Cases/)? Would really appreciate it because the time saved reproducing will be time spent fixing.

**Comment by jfillipe** [comment]
It´s not only this view, on events in Week View too. He show event time begin, **undefinedundefined** and title. 

:'(

I try everything e nothing.

**Comment by houseoftech** [comment]
Actually, my issue is being caused by the Vue wrapper, [vue-full-calendar](https://github.com/CroudSupport/vue-fullcalendar). It doesn't seem to matter which FullCalendar version is used, but after vue-full-calendar@2.1.1 it always produces the bug. If the fullcalendar package is removed from within vue-full-calendar and it uses your own version then there is no bug.

**Comment by jfillipe** [comment]
@arshaw I´ll simulate evening and will post here

**Comment by jfillipe** [comment]
@arshaw I´m trying to reproduce but i can´t 
:´(

**Comment by houseoftech** [comment]
@arshaw Are you using Vue or another framework where FullCalendar is wrapped in a component?

**Comment by arshaw** [comment]
seems like a vue-full-calendar specific problem. can someone post on their issue tracker? https://github.com/CroudSupport/vue-fullcalendar/issues
cc @BrockReece

**Comment by BrockReece** [comment]
Since v2.1.1 of vue-full-calendar, we froze our fullcalendar dependency version to 3.4.0 
This was mainly because one of my colleagues reported breaking changes between v3.4.0 and v3.5.0.

I have created a PR for vue-full-calendar which removes this freeze, do you want to see if my wrapper is still causing an issue?  https://github.com/CroudSupport/vue-fullcalendar/pull/74

**Comment by jfillipe** [comment]
But i´m not using Vue!! I´m using angular! Ever, and just now i´ve got this...

**Comment by kofarrell1076** [comment]
same issue here. Running MVC and Jquery. Only happens on second call to refresh fullcalendar. Fist time works ok.

**Comment by acerix** [comment]
It sounds like an issue with the wrapper you are using.  We need some way to reproduce the issue to look into it.

P.S. I'm working with @arshaw at fullcalendar, helping to triage tickets.

**Comment by kofarrell1076** [comment]
@acerix - I deleted my post as I sorted out the problem. Was an issue with moment.js. Fullcalendar works great now.
thanks

**Comment by charlwillia6** [comment]
This is still an issue even without vue.js and even if moment.js is in the head of the page if you are using ajax to load the page that FullCalendar will be rendered.  

I have FullCalendar (Scheduler to be specific) that loads in a tab.  When the tab is clicked, the htm for that tab is loaded by using ajax, and after the HTML loads and is injected (using `<div id=calendar></div>`, then I use a initialize_calendar() function to inject the calendar.  The issue here is that moment.js is still in the head of the page.  Yet FullCalendar is not seeing it, or something, because I get `26 FEBRUARY 2018UNDEFINEDUNDEFINEDUNDEFINEDUNDEFINED` in the title.  It is definitely an issue with moment.js and FullCalendar, because if I put $.getScript("moment.js"); above where I am calling the initialize_calendar function, then everything is fine.  But I use moment.js throughout my project, and I don't want to pull another instance of the moment.js file with each ajax call. 

So is there a way to initialize moment in my ajax call?  Why is it not recognizing that moment.js is already present?

**Comment by nc-csl** [comment]
It is definitely an bug between scheduler and moment. If I disable scheduler, everything works fine.

**Comment by osnersanchez** [comment]
I am using version 1.9.4 of fullcalendar-scheduler in angular and I also have that problem in the title

![image](https://user-images.githubusercontent.com/12916793/45598333-3d5dc580-b9a8-11e8-973f-6080b5706072.png)

I'm using the CodePen example

**Comment by MrMitch** [comment]
For anyone running into this, I had the issue with fullcalendar `3.10.0`.
I was using fullcalendar with webpack, and `import`ing the minified version of fullcalendar.
Using the non-minified version solved the problem.

**TL;DR :**

replace 
```js
import 'fullcalendar/dist/fullcalendar.min';
```

with 
```js
import 'fullcalendar/dist/fullcalendar';
```

**Comment by kudatr** [comment]
I had the same error, the reason I had it was i was importing both fullcalendar and fullcalendar-scheduler.  Importing one of the two and not both solved the issue.

`// import "fullcalendar"`
`import "fullcalendar-scheduler"`

**Comment by nil09** [comment]
@kudatr yes...! 

Do not import both calendar. 

```
// import "fullcalendar"
import "fullcalendar-scheduler"
```

**Comment by ThomasTheJohnson** [comment]
Had the same issue in an Angular project after updating packages, but there was a little more work. Had to make sure I was only using the fullcalendar-scheduler import in the .ts files and only importing fullcalendar-scheduler in my package.json since it automatically imports the supported version of fullcalendar with it. 

If you were using the fullcalendar.css you'll also have to change the path in your angular.json: ``node_modules/fullcalendar-scheduler/node_modules/fullcalendar/dist/fullcalendar.css`` instead of ``node_modules/fullcalendar/dist/fullcalendar.css``

**Comment by SumanBandopadhyay** [comment]
I am still facing the issue with undefined.
I can see multiple guidance and multiple ways to solve.

But I guess I am doing something wrong.
Please do let me know a way to solve the issue.

**Comment by kudatr** [comment]
@SumanBandopadhyay 

Did you make sure you are only importing one of the calendars? Don't import both fullcalendar and fullcalendar scheduler.  Only one of them.
